nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/CommentsMigrator.java

```diff
@@ -272,8 +272,11 @@ protected void migrateCommentsFromPropertyToSecured(CoreSession session, TreeCom
 
         // Strip ACLs
         ACP acp = session.getACP(commentIdRef);
-        acp.removeACL(LOCAL_ACL);
-        session.setACP(commentIdRef, acp, true);
+        // Case where a comment is on a placeless document which has no acp
+        if (acp != null) {
```

Comment:
https://jira.nuxeo.com/browse/NXP-28130 was supposed to fix this but I guess we forgot some cases :(